### ISSUE TITLE
Switch to cluster-wide operation (instead of single namespace)

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/drupal-operator#2]
See: https://github.com/operator-framework/operator-sdk/blob/master/doc/operator-scope.md#operator-scope

**Comment by geerlingguy** [comment]
Got through a bunch of little quirks (wasn't committing as often as I should've though!), and now hitting one final barrier:

```
    TASK [Create the drupal.drupal.org/v1alpha1.Drupal] ****************************
    changed: [localhost]
    
    TASK [Wait 5m for reconciliation to run] ***************************************
    FAILED - RETRYING: Wait 5m for reconciliation to run (50 retries left).
    ...
    FAILED - RETRYING: Wait 5m for reconciliation to run (1 retries left).
    fatal: [localhost]: FAILED! => {"attempts": 50, "changed": false, "resources": [{"apiVersion": "drupal.drupal.org/v1alpha1", "kind": "Drupal", "metadata": {"creationTimestamp": "2019-07-30T00:54:56Z", "generation": 1, "name": "example-drupal", "namespace": "example-drupal", "resourceVersion": "775", "selfLink": "/apis/drupal.drupal.org/v1alpha1/namespaces/example-drupal/drupals/example-drupal", "uid": "32094862-c5c3-42c7-896b-a944bf93f71d"}, "spec": {"database_image": "mariadb:10", "database_pvc_size": "1Gi", "drupal_hash_salt": "fe918c992fb1bcfa01f32303c8b21f3d0a0", "drupal_version": "8.7", "files_pvc_size": "1Gi", "manage_database": true}}]}
```

It seems like the operator might be getting stuck with one thing getting a `changed` over and over again, resulting in the playbook getting stuck in a reconciliation loop. I don't think it's the `drupal_hash_salt` (which was throwing me off earlier), just need more time to debug later.

**Comment by geerlingguy** [comment]
Hmm... everything works fine in Travis CI. Maybe letting my computer take a little sleep tonight will make it work locally too :P

**Comment by geerlingguy** [comment]
So this is fun... I kept getting "kubelet has disk pressure" warnings. At first I thought it was because the mounted `/build` volume had only like 20-25% of space remaining, but that didn't make too much sense because the K8s default is to taint the node if there's < 10% remaining.

So then I noticed after running DaisyDisk that my Docker VM image size was approaching 55 GB (and I had it configured to be maximum of 64 GB). That in itself shouldn't result in 90+% usage... but it seems it has. I resized the Docker VM image in the preferences and that triggered a delete/re-create (note this also wipes out all Docker images and containers... but I do that pretty much weekly anyways).

So now tests are running locally, yay!
